docs/workflows/ios-specific-workflow-steps/xcode-select.md

```diff
@@ -57,6 +57,20 @@ You can review the documentation for detailed information about the Xcode versio
 
 This step contains some input variable(s). It needs these variable(s) to work. The table below gives explanation for this variable(s).
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE5421-xcodeSelectInput.png' />
+
+:::info Manually Xcode Version Change
```

Comment:
IMO this info box should be added instead of following warn box:
https://capture.dropbox.com/GHaEGY6VwtYXa1XN